crates/uv/tests/pip_install.rs

```diff
@@ -1089,6 +1089,7 @@ fn install_editable_bare_cli() {
     ----- stderr -----
     Resolved 1 package in [TIME]
     Prepared 1 package in [TIME]
+    Installing to environment at [VENV]/bin/python3
```

Comment:
Shouldn't we be showing `[VENV]` not `[VENV]/bin/python3`? It seems a bit weird to show the path to the interpreter executable.

Reply:
Ah yeah, we could do that - something like:

```rust
        if is_outside_working_directory {
            if let Some(parent_dir) = python_executable.parent().and_then(|p| p.parent()) {
                writeln!(
                    printer.stderr(),
                    "{}",
                    format!(
                        "Installing to environment at {}",
                        parent_dir.user_display()
                    )
                        .dimmed()
                )?;
            }
        }
```
? This yields `[VENV]/` instead of `[VENV]/bin/python`

Reply:
I think you just want `PythonEnvironment::root()`, i.e., `venv.root()`?